students/shibin_mathew/lesson3/a_new_file.py

```diff
@@ -18,43 +18,44 @@ def exchange_first_last(seq):
 
```

Comment:
`seq[-1:] + seq[1:-1] + seq[:1]` <- the answer to exchange first last method which will work for list, string and a tuple